toolkit/README.md

```diff
@@ -68,10 +68,10 @@ pip3 install requests
 
 Run the script and pass in the two versions of Spin.
 
-**Note**: Use two number characters `vX.X` only i.e. `v2.6`:
+**Note**: Use two number characters `vX.X` only i.e. `v2.7`:
```

Comment:
We don't need to update the "example of a version number."  It will just get tedious.  If this bothers us then let's change it to something fantastical like "9.99" that clearly signals "this is not an actual version that needs to be maintained."